packages/site/src/components/Footer.js

```diff
@@ -17,7 +17,7 @@
 /* @flow */
 import React from 'react';
 import { createStyledComponent } from '@mineral-ui/component-utils';
-import Link from './Link';
+import Link from '@mineral-ui/link';
```

Comment:
Good thinking to update the site to use the new Link component.  Let's just try to keep this stuff as a separate commit.